developer_resources/developer_resources.md

```diff
@@ -38,4 +38,24 @@ Follow these steps to generate rdocs for now and existing SDK gems:
 
 Rdoc options are controlled with this [config file](https://github.com/urbanopt/urbanopt-scenario-gem/blob/develop/.rdoc_options).
 
+## Developing with the CLI
+
+When developing locally on URBANopt core gems and testing new functionality and dependencies via the CLI, it may be necessary to install local versions of core gems.  
+
+For local development, you will want to set the environment variable *FAVOR_LOCAL_GEMS* to 1. This enables local copies of gems in the Gemfile.  There are 2 Gemfiles of interest:
+
+1. When testing a new measure that will run in the OpenStudio workflow OSW file, enable the gem that contains the measure in the Gemfile of the project you are testing. You may have to modify the path or branch referenced to match what you are testing.
+
+1. When testing new functionality in the URBANOPT workflow (a new scenario post-processor, for example), enable the gem in the Gemfile of the URBANopt-cli local checkout. You may have to modify the path or branch referenced to match what you are testing.
+
+Follow these steps when testing local gems via the CLI:
+
+1. ```bundle install``` (to install the local gems that were just enabled in the Gemfile)
+1. ```bundle exec rake install``` (to install the CLI in a place that has access to those local gems)
+1. ```bundle exec uo ...``` (to execute the CLI calls)
+
+## Example Project Development
+
+1. Develop new functionality in the example project repo.  
```

Comment:
Do we want to call this a repo or just a directory/folder?

Reply:
directory!